components/steps/Status.tsx

```diff
@@ -23,10 +22,17 @@ const getStatusLabel = (status: string) => {
   return status[0].toUpperCase() + status.substring(1, status.length);
 };
 
-const getStage = (status: string): number => {
-  if (status === "unverified") {
+const getStage = (profile: Profile): number => {
+  if (profile.status === "unverified") {
     return 1;
-  } else if (status === "verified") {
+  } else if (profile.status === "verified") {
+    if (profile.submittedAt !== null) {
+      Sentry.captureMessage(
```

Comment:
Not sure which is the best Sentry method to use here but this should work?

Reply:
I think that seems good, maybe worth logging the profile object so we can get a bit more info, and so we can start digging through the db and stuff to find the issue?

Reply:
Good point, I’ll add profile in there